package.json

```diff
@@ -40,30 +40,29 @@
     "@angular/platform-browser": "7.2.15",
     "@angular/platform-browser-dynamic": "7.2.15",
     "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.17.0",
-    "@pact-foundation/pact": "9.0.0",
-    "@pact-foundation/pact-web": "9.0.0",
-    "@skyux-sdk/builder": "3.8.0",
+    "@blackbaud/auth-client": "2.17.2",
+    "@pact-foundation/pact": "9.2.0",
+    "@pact-foundation/pact-web": "9.2.0",
+    "@skyux-sdk/builder": "3.10.0",
     "@skyux-sdk/e2e": "3.1.2",
-    "@skyux-sdk/pact": "3.2.1",
     "@skyux-sdk/testing": "3.1.0",
     "@skyux/assets": "3.1.0",
-    "@skyux/config": "3.5.0",
-    "@skyux/core": "3.6.0",
-    "@skyux/http": "3.4.1",
-    "@skyux/i18n": "3.7.1",
-    "@skyux/indicators": "3.0.3",
-    "@skyux/modals": "3.0.6",
+    "@skyux/config": "3.6.0",
+    "@skyux/core": "3.7.0",
+    "@skyux/http": "3.7.0",
+    "@skyux/i18n": "3.9.0",
+    "@skyux/indicators": "3.1.0",
+    "@skyux/modals": "3.1.0",
     "@skyux/omnibar-interop": "3.1.0",
     "@skyux/router": "3.1.1",
-    "@skyux/theme": "3.7.0",
+    "@skyux/theme": "3.8.0",
     "@types/core-js": "2.5.2",
     "codelyzer": "4.5.0",
-    "core-js": "2.6.3",
-    "rxjs": "6.5.2",
-    "rxjs-compat": "6.5.2",
-    "ts-node": "8.3.0",
-    "tslint": "5.18.0",
+    "core-js": "2.6.9",
+    "rxjs": "6.5.3",
+    "rxjs-compat": "6.5.3",
+    "ts-node": "8.4.1",
+    "tslint": "5.20.0",
```

Comment:
Why so many package upgrades?

Reply:
Just development dependencies to fix some vulnerability warnings...

Reply:
👍